src/pages/Appointments/AppointmentsPage.tsx

```diff
@@ -242,13 +235,20 @@ function DateRangeDisplay({ dateFrom, dateTo }: DateRangeDisplayProps) {
 export default function AppointmentsPage(props: { facilityId?: string }) {
   const { t } = useTranslation();
   const authUser = useAuthUser();
+  const { qParams, updateQuery, resultsPerPage, Pagination } = useFilters({
+    limit: 15,
+  });
 
-  const [qParams, setQParams] = useQueryParams<QueryParams>();
-
+  const paginationComponent = (count: number) => {
+    return <Pagination totalCount={count} />;
+  };
   const facilityId = props.facilityId ?? authUser.home_facility!;
 
   const [activeTab, setActiveTab] = useView("appointments", "board");
 
+  const updateStatus = (value: string) =>
+    updateQuery({ ...qParams, status: value, page: null });
+
```

Comment:
is this needed? wouldn't use filters reset the page for you?

Reply:
then you can simply directly call updateQuery with status passed right? it'll handle the page reset right?